Controllers/TaskController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Controllers;
+
+use Models\TaskModel;
+
+class TaskController implements IController
+{
+    private $model;
+    private $fc;
```

Comment:
Name it more understandable? How can typical idiot understand wtf is going here?
Name it smth like **$frontController**
but don't **suka** use it here!